app/src/views/CookiePolicy/i18n.json

```diff
@@ -0,0 +1,70 @@
+{
+    "namespace": "cookiePolicy",
+    "strings": {
+        "cookiePolicyTitle": "Cookie Policy",
+        "disclaimerTitle": "Disclaimer",
+        "disclaimerDescription": "All the information contained in the web site is Copyright © The International Federation of Red Cross and Red Crescent Societies.All rights reserved. ",
+        "useOfOurInformationTitle": "Use Of Our Information",
+        "useOfOurInformationDescription1": "To use text information from this website, please obtain prior permission from the IFRC.",
+        "useOfOurInformationDescription2": "To use photographs from this website, please contact our {termsLink}.",
+        "useOfOurInformationAudiovisualLink": "audiovisual department",
+        "useOfOurInformationDescription3": "If you have any question about this website,or if you find any errors,please notify us.",
+        "useOfOurInformationDescriptionLink": "Click here for important information about fraudulent websites and emails.",
+        "ourPrivacyPolicyHeading": "Our Privacy Policy",
+        "ourPrivacyPolicyContent": "This website is operated by the Federation. The Federation takes your privacy and the security of our personal information-meaning and information relating to an identified or identifiable individual- seriously.Please take a moment to read this privacy and learn  about the Federation's general privacy and information security principles.Any changes to this Privacy Policy will be posted on this page.The version of the policy was published on {publishedDay} {publishedDate} {publishedYear}",
+        "dataCollectedByAccessingHeading": "A) Data collected by accessing our sites",
+        "informationProvideHeading": "1. Information you provide",
```

Comment:
we use list tag in informationProvideheadings?

Reply:
We are using containers here, so lets skip this for now.